lib/src/dynamics/body.dart

```diff
@@ -25,8 +25,9 @@ class Body {
   /// The swept motion for CCD
   final Sweep _sweep = Sweep();
 
-  /// the linear velocity of the center of mass
-  final Vector2 _linearVelocity = Vector2.zero();
+  /// The linear velocity of the center of mass. Do not modify directly, instead
+  /// use setLinearVelocity, applyLinearImpulse or applyForce
+  final Vector2 linearVelocity = Vector2.zero();
   double _angularVelocity = 0.0;
```

Comment:
whats the rationale of changing linVelocity to be public and angularVelocity not?

Reply:
Since it is used so heavily we don't want to clone it in a getter for performance reasons, and if we just had a getter returning the private field it could still be modified by the user. There should be a getter for `angularVelocity` though.

Reply:
Aah, apparently it was just far down there in the middle.